### PR TITLE
Adding some Azure Batch related content from docs.microsoft.com

### PR DESCRIPTION
**Pull request by heoelri**
Summary of changes proposed in this pull request:

- This PR contains some Azure Batch related content from docs.microsoft.com

> IMPORTANT: Make sure you are familiar with the [Contributing](/Contributing.md) guidance. Particularly, **don't edit \*.md files directly - make your changes in the source \*.json files**.

### CODE REVIEW

File: data/servicelens-compute.data.json [hunk 1]
```diff
@@ -1146,5 +1146,92 @@
...
+  {
+    "type": "Configuration Recommendations",
+    "pillars": [
+      "reliability",
+      "operationalexcellence"
+    ],
+    "lens": "service",
+    "category": "Compute",
+    "subCategory": "Azure Batch",
+    "title": "Pre-create all required accounts in each region, such as the Batch account and storage account.",
```
Comment (sebader) [inline data/servicelens-compute.data.json:1171]:
```suggestion
    "title": "Pre-create all required services in each region, such as the Batch account and storage account.",
```

File: data/servicelens-compute.data.json [hunk 2]
```diff
@@ -1146,5 +1146,92 @@
...
+    "context": "There is often no charge for having accounts created, and charges accrue only when the account is used or when data is stored.",
```
Comment (sebader) [inline data/servicelens-compute.data.json:1172]:
```suggestion
    "context": "There is often no charge for having accounts created, and charges occur only when the account is used or when data is stored.",
```

File: data/servicelens-compute.data.json [hunk 3]
```diff
@@ -1146,5 +1146,92 @@
...
+  {
+    "type": "Configuration Recommendations",
+    "pillars": [
+      "reliability",
+      "operationalexcellence"
+    ],
+    "lens": "service",
+    "category": "Compute",
+    "subCategory": "Azure Batch",
+    "title": "Make sure the appropriate quotas are set on all accounts ahead of time, so you can allocate the required number of cores using the Batch account.",
```
Comment (sebader) [inline data/servicelens-compute.data.json:1184]:
```suggestion
    "title": "Make sure the appropriate quotas are set on all subscriptions ahead of time, so you can allocate the required number of cores using the Batch account.",
```

File: data/servicelens-compute.data.json [hunk 4]
```diff
@@ -1146,5 +1146,93 @@
...
+  {
+    "type": "Design Considerations",
+    "pillars": [
+      "reliability",
+      "operationalexcellence"
+    ],
+    "lens": "service",
+    "category": "Compute",
+    "subCategory": "Azure Batch",
+    "title": "Use templates and/or scripts to automate the deployment of the application in a region.",
```
Comment (sebader) [inline data/servicelens-compute.data.json:1196]: isnt that a more general recommendation: use IaC for everything. why mention it for Batch?

**Review by sebader — CHANGES_REQUESTED** [review]

File: data/servicelens-compute.data.json (hunk 2 above)
Comment (heoelri) [reply data/servicelens-compute.data.json:1172]: I looked this up and it seems like "accrue" is valid here.

File: data/servicelens-compute.data.json (hunk 4 above)
Comment (heoelri) [reply data/servicelens-compute.data.json:1196]: dropped

File: data/servicelens-compute.data.json (hunk 2 above)
Comment (sebader) [reply data/servicelens-compute.data.json:1172]: ok

**Review by sebader — APPROVED** [review]
LGTM

**Pull request merged**